sqlconnect/internal/redshift/mappings.go

```diff
@@ -44,6 +44,10 @@ func jsonRowMapper(databaseTypeName string, value any) any {
 			if n, err := strconv.ParseFloat(string(v), 64); err == nil {
 				return n
 			}
+		case string:
```

Comment:
**Note:** sdk driver returns `NUMERIC` data types as `string` instead of `[]byte`